cache/modules/cloudfront_policies/cache_policies.tf

```diff
@@ -7,6 +7,12 @@ resource "aws_cloudfront_cache_policy" "toggle_cookies_only" {
   max_ttl     = local.one_day
 
   parameters_in_cache_key_and_forwarded_to_origin {
+    // https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/controlling-the-cache-key.html#cache-policy-compressed-objects
+    // This needs to be here as, despite what the AWS docs say, adding Accept-Encoding
+    // to the headers whitelist in a request policy fails with the grammatically questionable error:
+    // The parameter Headers contains Accept-Encoding that is not allowed.
```

Comment:
Very minor: let's indent this line to make it clear you're quoting vs writing free text.